react/components/atoms/lightbox/lightbox.js

```diff
@@ -106,20 +317,45 @@ export class Lightbox extends PureComponent {
                     visible={this.state.visible}
                     onRequestClose={this.onBackButtonPress}
                 >
-                    <View style={styles.fullscreenContainer}>
-                        <Image style={this._imageFullscreenStyle()} source={this._imageSource()} />
-                        <ButtonIcon
-                            icon={"close"}
-                            onPress={this.onClosePress}
-                            style={styles.buttonClose}
-                            iconStrokeWidth={2}
-                            size={isTabletSize() ? 52 : 34}
-                            iconHeight={isTabletSize() ? 34 : 22}
-                            iconWidth={isTabletSize() ? 34 : 22}
-                            backgroundColor={"#000000"}
-                            iconStrokeColor={"#ffffff"}
-                        />
-                    </View>
+                    <GestureHandlerRootView style={styles.gestureHandlerRootView}>

```

Comment:
Regarding what they say in their documentation, this should wrap the entire library code or we need to put it in the Robin's and other apps `app.js` https://docs.swmansion.com/react-native-gesture-handler/docs/#js